### PR TITLE
Portfolio - mobile menu

### PR DESCRIPTION
**Pull request by isaacpitwa**
## Mobile Menu

1. Add Javascript file `index.js`
2. Add Mobile Menu  view + its  styles
3. function to add styles to mobile menu  using `onclick` event 
4. fix all lint errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,15 +25,25 @@ <h1 class="nav-brand">My Logo</h1>
...
-                <li class="nav-item"><a href="#" >About</a></li>
-                <li class="nav-item"><a href="#" >Contact</a></li>
+                <li class="nav-item"><a href="#">Portoflio</a></li>
+                <li class="nav-item"><a href="#">About</a></li>
+                <li class="nav-item"><a href="#">Contact</a></li>
             </ul>
         </nav>
-        <img src="images/icons/Union.svg" alt="Menu" />
+        <div id="toggle-menu" class="hidden">
+            <img src="images/icons/close.svg" alt="Close"  onclick="toggleMenu(this)"/>
```
Comment (mateo951) [inline index.html]: Kindly make sure that your button functionality works 😥 When being clicked, the menu is not being shown.  The issue is the following:

image

File: index.html [hunk 2]
```diff
@@ -6,7 +6,16 @@
...
+    <title>Isaac Pitwa Portoflio</title>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-ES0LN8KC8W"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag() { dataLayer.push(arguments); }
+        gtag('js', new Date());
+
+        gtag('config', 'G-ES0LN8KC8W');
+    </script>
```
Comment (mateo951) [inline index.html]: In order for us to follow [JS Best Practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) is important to keep all the JavaScript code in JS files, not in the script tag. Kindly add this functionality into a file to meet with these practices and the project requirements 😁

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by mateo951** [comment]
Please be sure to add JS linters to your project. Follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) to do so 😉

File: index.html [hunk 3]
```diff
@@ -16,15 +18,25 @@ <h1 class="nav-brand">My Logo</h1>
...
-        <img src="images/icons/Union.svg" alt="Menu" />
+        <div id="toggle-menu" class="hidden">
+            <img src="images/icons/close.svg" alt="Close"  onclick="toggleMenu()"/>
+            <nav class="toggle-nav">
+                <ul class="nav-list">
+                    <li class="nav-item"><a href="#" onclick="mobileMenuOption(event,'Portoflio')">Portoflio</a></li>
+                    <li class="nav-item"><a href="#" onclick="mobileMenuOption(event,'About')">About</a></li>
+                    <li class="nav-item"><a href="#" onclick="mobileMenuOption(event,'Contact')">Contact</a></li>
+                </ul>
+            </nav>
```
Comment (FrederickMih) [inline index.html]: - [ ] Hi @isaacpitwa, nice job :+1:  for fixing the previous issue highlighted by the previous code reviewer. Once more, your close button is not visible. See the screenshots below:

Your Implementation | Figma Design
:---:|:---:
 | 

Please also see into the spacings of the menu options.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @isaacpitwa , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @isaacpitwa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**